src/Commands/DataExplorer/TableSchemaCommand.cs

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using AzureMcp.Arguments.DataExplorer;
+using AzureMcp.Models.Argument;
+using AzureMcp.Models.Command;
+using AzureMcp.Services.Interfaces;
+using Microsoft.Extensions.Logging;
+using ModelContextProtocol.Server;
+using System.CommandLine.Parsing;
+using System.Text.Json;
+
+namespace AzureMcp.Commands.DataExplorer;
+
+public sealed class TableSchemaCommand(ILogger<TableSchemaCommand> logger) : BaseTableCommand<TableSchemaArguments>
+{
+    private readonly ILogger<TableSchemaCommand> _logger = logger;
+
+    protected override string GetCommandName() => "schema";
+
+    protected override string GetCommandDescription() =>
+        "Get the schema of a specific table in an Azure Data Explorer (Kusto) database.";
```

Comment:
depending on what we decide, preferably also add what is the expected return type and maybe an example